app/bundles/ChannelBundle/Model/MessageQueueModel.php

```diff
@@ -136,6 +140,68 @@ public function processFrequencyRules(
     }
 
     /**
+     * Adds messages to the queue
```

Comment:
```suggestion
     * Adds messages to the queue.
```
CS check is failing because of this.

Reply:
@anton-vlasenko can you please? I don't have rights on the source repo to commit instead of you.